site/src/pages/components/customizableselect.mdx

```diff
@@ -177,14 +170,12 @@ This example uses the customizable `<select>` element with custom CSS to target
   <button>
     <selectedoption></selectedoption>
   </button>
-  <datalist>
```

Comment:
Is the `<datalist>` still allowed? If so, want to keep one example with it? Somewhere, not necessarily this one.

Reply:
Yes, it's still allowed, and supporting it imposes quite a lot of complexity. I think I'm going to have to work some more on justifying it.

So far I can think of 3 reasons:
1. If we reinstate the `<input>` creating a `</select>`, then `<datalist>` would be needed in order to support `<input>`. Unfortunately `<input>` is not part of the content model and something which we can make accessible yet, so this isn't a great argument.
2. In order to make a pseudo element which targets the shadowroot `<datalist>` as capable as the element itself, we will probably have to do a bunch of work in CSS in order to make sure that all pseudo-elements can be used on it among other things.
3. View transitions might need an actual javascript handle to the element, but I'm not sure. There could also be other cases, like if the author wanted to show or hide the popover from script for some reason.

Reply:
I'm following up with adam to see if we can get a view transitions demo which I could put in the explainer, which would require and justify `<datalist>`